### DISCUSSION TITLE
Active keka window not on top

### DISCUSSION THREAD

**Discussion by kw-sudo** [aonez/Keka#1222] [Q&A] [answered]
Hi,

If I want to compress or unpack files (extract/compress to...) via the Finder context menu (macOS 13.2.1 Ventura), the selection window no longer appears directly in the foreground to move on. Not even via task switch.

I always have to press the F3 (desktop overview) key to pick and bring it up.

![keka](https://user-images.githubusercontent.com/104136223/219319848-59a52f42-f0cc-4ef0-8461-62b95c178a16.jpg)

Regards,
Oliver

**Comment by aonez** [comment, marked as answer]
Thanks a lot for the feedback @master-kw.

I've just uploaded a test build fixing this issue, in case you want to test it: [Keka-v1.2.61.r5109](https://github.com/aonez/Keka/releases/download/dev-test-builds/Keka-v1.2.61.r5109.7z)

Will be officially released most probably the next Thursday.

**Reply by kw-sudo** [reply]
Hi aonez,

thank you for the test build...you made my day! With this latest version it works (again).
